--- original
+++ modified
@@ -8,12 +8,6 @@
 		<AffectsSavedGames>0</AffectsSavedGames>
 		<Version>1.0.4</Version>
 		<URL>https://forums.civfanatics.com/resources/leonardfactorys-policy-yield-previews.32012/</URL>
-	<!-- Added automatically by CivMods: Do not edit -->
-	<CivModsCategory>UI Mods</CivModsCategory>
-	<CivModsInternalVersionId>lxmi4589x4pn7c8</CivModsInternalVersionId>
-	<CivModsURL>https://forums.civfanatics.com/resources/leonardfactorys-policy-yield-previews.32012/</CivModsURL>
-	<CivModsVersion>1.0.4</CivModsVersion>
-	<CivModsVersionDate>2025-09-01T22:00:38.000Z</CivModsVersionDate>
 	</Properties>
 	<Dependencies>
 		<Mod id="base-standard" title="LOC_MODULE_BASE_STANDARD_NAME" />
