--- original
+++ modified
@@ -7,12 +7,6 @@
 		<Authors>Dunkleosteus</Authors>
 		<Package>Mod</Package>
 		<AffectsSavedGames>0</AffectsSavedGames>
-	<!-- Added automatically by CivMods: Do not edit -->
-	<CivModsCategory>Misc</CivModsCategory>
-	<CivModsInternalVersionId>9yf1k18d43i0tvp</CivModsInternalVersionId>
-	<CivModsURL>https://forums.civfanatics.com/resources/no-age-progress-from-future-tech-and-future-civic.32062/</CivModsURL>
-	<CivModsVersion>1.1</CivModsVersion>
-	<CivModsVersionDate>2025-04-03T21:36:54.000Z</CivModsVersionDate>
 	</Properties>
 	<Dependencies>
 		<Mod id="base-standard" title="LOC_MODULE_BASE_STANDARD_NAME"/>
