--- original
+++ modified
@@ -7,12 +7,6 @@
 		<Authors>Mallek (Based on original work by Yamada)</Authors>
 		<Package>Mod</Package>
 		<AffectsSavedGames>0</AffectsSavedGames>
-	<!-- Added automatically by CivMods: Do not edit -->
-	<CivModsCategory>UI Mods</CivModsCategory>
-	<CivModsInternalVersionId>q34n053kybat166</CivModsInternalVersionId>
-	<CivModsURL>https://forums.civfanatics.com/resources/enhanced-town-focus-info.31969/</CivModsURL>
-	<CivModsVersion>v1.1.10</CivModsVersion>
-	<CivModsVersionDate>2025-04-30T22:27:08.000Z</CivModsVersionDate>
 	</Properties>
 	<Dependencies>
 		<Mod id="base-standard" title="LOC_MODULE_BASE_STANDARD_NAME" />
