--- original
+++ modified
@@ -7,12 +7,6 @@
 		<Authors>Cyberdisc</Authors>
 		<Package>Mod</Package>
 		<AffectsSavedGames>0</AffectsSavedGames>
-	<!-- Added automatically by CivMods: Do not edit -->
-	<CivModsCategory>UI Mods</CivModsCategory>
-	<CivModsInternalVersionId>0i891583w114s2s</CivModsInternalVersionId>
-	<CivModsURL>https://forums.civfanatics.com/resources/better-pause-menu.31873/</CivModsURL>
-	<CivModsVersion>3</CivModsVersion>
-	<CivModsVersionDate>2025-03-30T00:21:47.000Z</CivModsVersionDate>
 	</Properties>
 	<Dependencies>
 		<Mod id="base-standard" title="LOC_MODULE_BASE_STANDARD_NAME" />
