--- original
+++ modified
@@ -7,12 +7,6 @@
 		<Authors>Cyberdisc</Authors>
 		<Package>Mod</Package>
 		<AffectsSavedGames>0</AffectsSavedGames>
-	<!-- Added automatically by CivMods: Do not edit -->
-	<CivModsCategory>UI Mods</CivModsCategory>
-	<CivModsInternalVersionId>42z5u1jd143002i</CivModsInternalVersionId>
-	<CivModsURL>https://forums.civfanatics.com/resources/chromatic-fow-resources.32130/</CivModsURL>
-	<CivModsVersion>3</CivModsVersion>
-	<CivModsVersionDate>2025-06-27T22:55:38.000Z</CivModsVersionDate>
 	</Properties>
 	<Dependencies>
 		<Mod id="base-standard" title="LOC_MODULE_BASE_STANDARD_NAME"/>
