--- original
+++ modified
@@ -9,12 +9,6 @@
         <URL>https://forums.civfanatics.com/resources/resource-re-sorts.32032/</URL>
         <Package>Mod</Package>
         <AffectsSavedGames>0</AffectsSavedGames>
-    <!-- Added automatically by CivMods: Do not edit -->
-    <CivModsCategory>UI Mods</CivModsCategory>
-    <CivModsInternalVersionId>k332225yu396y98</CivModsInternalVersionId>
-    <CivModsURL>https://forums.civfanatics.com/resources/resource-re-sorts.32032/</CivModsURL>
-    <CivModsVersion>2.2.2</CivModsVersion>
-    <CivModsVersionDate>2025-08-20T03:57:55.000Z</CivModsVersionDate>
     </Properties>
     <Dependencies>
         <Mod id="base-standard" title="LOC_MODULE_BASE_STANDARD_NAME" />
