--- original
+++ modified
@@ -7,12 +7,6 @@
         <Package>Mod</Package>
         <AffectsSavedGames>0</AffectsSavedGames>
         <Version>1.1.0</Version>
-    <!-- Added automatically by CivMods: Do not edit -->
-    <CivModsCategory>UI Mods</CivModsCategory>
-    <CivModsInternalVersionId>q930vus5b9yj2wp</CivModsInternalVersionId>
-    <CivModsURL>https://forums.civfanatics.com/resources/treasure-lens.31942/</CivModsURL>
-    <CivModsVersion>v1.1.0</CivModsVersion>
-    <CivModsVersionDate>2025-03-13T01:03:38.000Z</CivModsVersionDate>
     </Properties>
     <Dependencies>
         <Mod id="base-standard" title="LOC_MODULE_BASE_STANDARD_NAME" />
