--- original
+++ modified
@@ -9,12 +9,6 @@
         <URL>https://forums.civfanatics.com/resources/city-hall.31946/</URL>
         <Package>Mod</Package>
         <AffectsSavedGames>0</AffectsSavedGames>
-    <!-- Added automatically by CivMods: Do not edit -->
-    <CivModsCategory>UI Mods</CivModsCategory>
-    <CivModsInternalVersionId>5g996f0uj14wgyh</CivModsInternalVersionId>
-    <CivModsURL>https://forums.civfanatics.com/resources/city-hall.31946/</CivModsURL>
-    <CivModsVersion>2.0.3</CivModsVersion>
-    <CivModsVersionDate>2025-08-28T21:16:56.000Z</CivModsVersionDate>
     </Properties>
     <Dependencies>
         <Mod id="base-standard" title="LOC_MODULE_BASE_STANDARD_NAME" />
