--- original
+++ modified
@@ -9,12 +9,6 @@
         <URL>https://forums.civfanatics.com/resources/31961/</URL>
         <Package>Mod</Package>
         <AffectsSavedGames>0</AffectsSavedGames>
-    <!-- Added automatically by CivMods: Do not edit -->
-    <CivModsCategory>UI Mods</CivModsCategory>
-    <CivModsInternalVersionId>96970ql652zm77q</CivModsInternalVersionId>
-    <CivModsURL>https://forums.civfanatics.com/resources/f1rstdans-cool-ui.31961/</CivModsURL>
-    <CivModsVersion>1.9.1</CivModsVersion>
-    <CivModsVersionDate>2025-06-28T08:21:32.000Z</CivModsVersionDate>
     </Properties>
     <Dependencies>
         <Mod id="base-standard" title="LOC_MODULE_BASE_STANDARD_NAME" />
