--- original
+++ modified
@@ -6,12 +6,6 @@
         <Authors>wltk</Authors>
         <Package>Mod</Package>
         <AffectsSavedGames>0</AffectsSavedGames>
-    <!-- Added automatically by CivMods: Do not edit -->
-    <CivModsCategory>UI Mods</CivModsCategory>
-    <CivModsInternalVersionId>a8y97q5ofbyxw89</CivModsInternalVersionId>
-    <CivModsURL>https://forums.civfanatics.com/resources/detailed-tech-civic-progress.31924/</CivModsURL>
-    <CivModsVersion>1.0.3</CivModsVersion>
-    <CivModsVersionDate>2025-05-03T17:23:37.000Z</CivModsVersionDate>
     </Properties>
     <Dependencies>
         <Mod id="base-standard" title="LOC_MODULE_BASE_STANDARD_NAME" />
