--- original
+++ modified
@@ -7,12 +7,6 @@
 		<Authors>Slothoth</Authors>
 		<Package>Mod</Package>
 		<AffectsSavedGames>0</AffectsSavedGames>
-	<!-- Added automatically by CivMods: Do not edit -->
-	<CivModsCategory>Mod Components</CivModsCategory>
-	<CivModsInternalVersionId>4dgt61uw9j43g94</CivModsInternalVersionId>
-	<CivModsURL>https://forums.civfanatics.com/resources/fealty-transfer-settlements-to-allies.32309/</CivModsURL>
-	<CivModsVersion>1.0</CivModsVersion>
-	<CivModsVersionDate>2025-05-10T03:49:46.000Z</CivModsVersionDate>
 	</Properties>
 	<Dependencies>
 		<Mod id="base-standard" title="LOC_MODULE_BASE_STANDARD_NAME"/>
