--- original
+++ modified
@@ -7,12 +7,6 @@
 		<Authors>Cyberdisc</Authors>
 		<Package>Mod</Package>
 		<AffectsSavedGames>0</AffectsSavedGames>
-	<!-- Added automatically by CivMods: Do not edit -->
-	<CivModsCategory>UI Mods</CivModsCategory>
-	<CivModsInternalVersionId>121jvo81ajryi91</CivModsInternalVersionId>
-	<CivModsURL>https://forums.civfanatics.com/resources/immersive-diplomacy-screen.31919/</CivModsURL>
-	<CivModsVersion>2</CivModsVersion>
-	<CivModsVersionDate>2025-06-26T23:30:53.000Z</CivModsVersionDate>
 	</Properties>
 	<Dependencies>
 		<Mod id="base-standard" title="LOC_MODULE_BASE_STANDARD_NAME" />
