--- original
+++ modified
@@ -6,12 +6,6 @@
         <Authors>wltk</Authors>
         <Package>Mod</Package>
         <AffectsSavedGames>0</AffectsSavedGames>
-    <!-- Added automatically by CivMods: Do not edit -->
-    <CivModsCategory>UI Mods</CivModsCategory>
-    <CivModsInternalVersionId>r1ka512hc5m0c8w</CivModsInternalVersionId>
-    <CivModsURL>https://forums.civfanatics.com/resources/wltks-detailed-map-tacks.32126/</CivModsURL>
-    <CivModsVersion>1.1.1</CivModsVersion>
-    <CivModsVersionDate>2025-05-03T17:27:30.000Z</CivModsVersionDate>
     </Properties>
     <Dependencies>
         <Mod id="base-standard" title="LOC_MODULE_BASE_STANDARD_NAME" />
