--- original
+++ modified
@@ -8,12 +8,6 @@
 		<Package>Mod</Package>
 		<AffectsSavedGames>0</AffectsSavedGames>
 		<Version>2.4</Version>
-	<!-- Added automatically by CivMods: Do not edit -->
-	<CivModsCategory>UI Mods</CivModsCategory>
-	<CivModsInternalVersionId>375d7a8w6z8q13x</CivModsInternalVersionId>
-	<CivModsURL>https://forums.civfanatics.com/resources/mantismaestros-compact-production-chooser.31862/</CivModsURL>
-	<CivModsVersion>2.4</CivModsVersion>
-	<CivModsVersionDate>2025-06-23T17:37:50.000Z</CivModsVersionDate>
 	</Properties>
 	<Dependencies>
 		<Mod id="base-standard" title="LOC_MODULE_BASE_STANDARD_NAME"/>
