--- original
+++ modified
@@ -6,12 +6,6 @@
         <Authors>nasuellia</Authors>
         <Package>Mod</Package>
         <AffectsSavedGames>0</AffectsSavedGames>
-    <!-- Added automatically by CivMods: Do not edit -->
-    <CivModsCategory>UI Mods</CivModsCategory>
-    <CivModsInternalVersionId>0kpww6hz59c7rt3</CivModsInternalVersionId>
-    <CivModsURL>https://forums.civfanatics.com/resources/nasuellia-non-sticky-selection.31966/</CivModsURL>
-    <CivModsVersion>0.9.9</CivModsVersion>
-    <CivModsVersionDate>2025-04-24T16:54:59.000Z</CivModsVersionDate>
     </Properties>
     <Dependencies>
         <Mod id="core" title="LOC_MODULE_CORE_NAME"/>
