--- original
+++ modified
@@ -8,12 +8,6 @@
 		<Package>Mod</Package>
 		<AffectsSavedGames>1</AffectsSavedGames>
 		<ShowInBrowser>1</ShowInBrowser>
-	<!-- Added automatically by CivMods: Do not edit -->
-	<CivModsCategory>Misc</CivModsCategory>
-	<CivModsInternalVersionId>7b7fa89s9t9sf1q</CivModsInternalVersionId>
-	<CivModsURL>https://forums.civfanatics.com/resources/rhq-artificially-intelligent-ai-mod.31881/</CivModsURL>
-	<CivModsVersion>RHQ AI Mod 2.07</CivModsVersion>
-	<CivModsVersionDate>2025-05-02T23:23:14.000Z</CivModsVersionDate>
 	</Properties>
 	<Dependencies>
 		<Mod id="base-standard" title="LOC_MODULE_BASE_STANDARD_NAME"/>
