--- original
+++ modified
@@ -7,12 +7,6 @@
         <Authors>Moxl</Authors>
         <Package>Mod</Package>
         <AffectsSavedGames>0</AffectsSavedGames>
-    <!-- Added automatically by CivMods: Do not edit -->
-    <CivModsCategory>UI Mods</CivModsCategory>
-    <CivModsInternalVersionId>w6vb6l597i2e6yv</CivModsInternalVersionId>
-    <CivModsURL>https://forums.civfanatics.com/resources/moxl-map-search.31889/</CivModsURL>
-    <CivModsVersion>1.2</CivModsVersion>
-    <CivModsVersionDate>2025-03-10T09:29:16.000Z</CivModsVersionDate>
     </Properties>
     <Dependencies>
         <Mod id="base-standard" title="LOC_MODULE_BASE_STANDARD_NAME"/>
