--- original
+++ modified
@@ -6,12 +6,6 @@
         <Authors>nasuellia</Authors>
         <Package>Mod</Package>
         <AffectsSavedGames>0</AffectsSavedGames>
-    <!-- Added automatically by CivMods: Do not edit -->
-    <CivModsCategory>UI Mods</CivModsCategory>
-    <CivModsInternalVersionId>515p4i7z4bah5p3</CivModsInternalVersionId>
-    <CivModsURL>https://forums.civfanatics.com/resources/nasuellia-non-sticky-selection.31966/</CivModsURL>
-    <CivModsVersion>v1.0.1</CivModsVersion>
-    <CivModsVersionDate>2025-06-24T14:05:37.000Z</CivModsVersionDate>
     </Properties>
     <Dependencies>
         <Mod id="core" title="LOC_MODULE_CORE_NAME"/>
